readme.txt

```diff
@@ -4,8 +4,8 @@ Contributors: elemntor, KingYes, ariel.k, jzaltzberg, mati1000, bainternet
 Requires at least: 5.9
 Tested up to: 6.2
 Stable tag: 1.0.0
-Version: 1.0.1
```

Comment:
Please change the version to the new one (1.1.0) because it will can break website